platform-hub-api/app/services/kubernetes/cluster_service.rb

```diff
@@ -2,83 +2,50 @@ module Kubernetes
   module ClusterService
```

Comment:
Do we need this service anymore? For "regular" AR models we just tend to call the basic CRUD operations directly in the controllers, as opposed to wrapping up the logic in services like this. (e.g. `ProjectsController`). May be more consistent to do it that way?

Note: it's probably a good idea to wrap any non-standard CRUD logic in services like this, but it doesn't look like we do anything out of the ordinary here?

Reply:
Also, this applies to the fetch/querying methods too – seems we should just be able to use the model directly in the controller?